Misc/NEWS.d/next/Tools-Demos/2023-08-25-22-40-12.gh-issue-108494.4RbDdu.rst

```diff
@@ -1,2 +1,3 @@
 :ref:`Argument Clinic <howto-clinic>` now has a partial support of the
-:ref:`Limited API <limited-c-api>`. Patch by Victor Stinner.
+:ref:`Limited API <limited-c-api>`: see Argument Clinic :ref:`How to use the
+Limited C API <clinic-howto-limited-capi>`. Patch by Victor Stinner.
```

Comment:
```suggestion
:ref:`Limited API <limited-c-api>`: see :ref:`clinic-howto-limited-capi`.
Patch by Victor Stinner.
```

Reply:
We usually do not include names of core developers in NEWS entry.